lib/child-writer/index.js

```diff
@@ -139,6 +139,8 @@ exports.write = (image, drive, options) => {
       // to provide better encapsulation.
       if (messageCommand === 'error') {
         emitError(robot.recomposeErrorMessage(parsedMessage));
+      } else if (messageCommand === 'log') {
+        console.log(message.data);
```

Comment:
Should this be `messageData`? Or `messageData.data`? Or is `message.data` indeed correct??
I think I need to go and lay down, I'm getting dizzy ;-)